src/main/java/org/veupathdb/service/eda/ds/plugin/alphadiv/AlphaDivScatterplotPlugin.java

```diff
@@ -134,7 +134,7 @@ protected void writeResults(OutputStream out, Map<String, InputStream> dataStrea
           DEFAULT_SINGLE_STREAM_NAME + 
        ", by=" + computeEntityIdColName +")");
       connection.voidEval(getVoidEvalVarMetadataMap(DEFAULT_SINGLE_STREAM_NAME, varMap));
-      connection.voidEval("map <- rbind(map, list('id'=attributes(alphaDivDT)$computedVariableDetails$variableId," +
+      connection.voidEval("map <- rbind(map, list('id'=veupathUtils::toColNameOrNull(attributes(alphaDivDT)$computedVariableDetails)," +
                                                  "'plotRef'='yAxisVariable'," +
                                                  "'dataType'=attributes(alphaDivDT)$computedVariableDetails$dataType," +
                                                  "'dataShape'=attributes(alphaDivDT)$computedVariableDetails$dataShape," +
```

Comment:
Note alphaDiv and betaDiv are also returning displayLabels in that computedVariableDetails list.

Reply:
`attributes(dt)$computedVariableDetails$displayLabel`